apps/checker/handlers/checker.go

```diff
@@ -302,5 +307,5 @@ func (h Handler) HTTPCheckerHandler(c *gin.Context) {
 
 	}
 
-	c.JSON(http.StatusOK, gin.H{"message": "ok"})
```

Comment:
@mxkaske  we used to return something dummy 
I'm returning the result of the ping instead 
This is useful to get instant result in the /run endoint